api/converter_utils.go

```diff
@@ -721,7 +721,7 @@ func signedTxnWithAdToTransaction(stxn *sdk.SignedTxnWithAD, extra rowData) (gen
 		keys := make([]tuple, 0)
 
 		for k := range stxn.ApplyData.EvalDelta.LocalDeltas {
-			addr, err := edIndexToAddress(k, stxn.Txn, stxn.ApplyData.EvalDelta.SharedAccts)
+			addr, err := edIndexToAddress(k, stxn.Txn, stxn.EvalDelta.SharedAccts)
```

Comment:
Similarly, I think this is bad, you should know it's ApplyData.EvalDelta

Reply:
On this one, I probably wouldn't comment on this regardless of which way the author originally wrote it, but I agree we should not force the author not to use `ApplyData`.